Common/Days.swift

```diff
@@ -88,3 +88,29 @@ public struct Days: OptionSet, Codable {
         return self.contains(Days.allCases[gregorianWeekdayIndex-1])
     }
 }
+    
+extension Days {
+    static public func combineOptions(_ expandedDays: Set<Days>) -> Days {
+        var combinedDays = Days.none
+        expandedDays.forEach { day in
+            combinedDays.insert(day)
+        }
+        return combinedDays
```

Comment:
Consider avoiding a mutable local var by instead using a more functional approach:
```suggestion
    return expandedDays.reduce(Days.none) {
            return $1.union($0)
        }
```

Reply:
like it. thanks